crates/swc_css_ast/src/selector.rs

```diff
@@ -197,34 +197,32 @@ pub struct PseudoElementSelector {
 
 #[ast_node]
 pub enum PseudoSelectorChildren {
-    #[tag("Nth")]
-    Nth(Nth),
+    #[tag("AnPlusB")]
+    AnPlusB(AnPlusB),
+
+    #[tag("Ident")]
+    Ident(Ident),
+
+    #[tag("SelectorList")]
+    SelectorList(SelectorList),
 
     #[tag("TokenAndSpan")]
     PreservedToken(TokenAndSpan),
 }
 
-#[ast_node("Nth")]
-pub struct Nth {
-    pub span: Span,
-    pub nth: NthValue,
-    pub selector_list: Option<SelectorList>,
+#[ast_node]
+pub enum AnPlusB {
+    #[tag("Ident")]
+    Ident(Ident),
```

Comment:
Why is `Ident` variant required for `2n + 1` ?

Reply:
`An+B` can `even`/`odd` - https://www.w3.org/TR/css-syntax-3/#the-anb-type, so it can be as notation and can be ident